Demo/TestController.cs

```diff
@@ -74,7 +75,7 @@ public JsonResult MakeCredentialOptionsTest([FromBody] TEST_MakeCredentialParams
         var options = _fido2.RequestNewCredential(user, existingKeys, opts.AuthenticatorSelection, opts.Attestation, exts);
 
         // 4. Temporarily store options, session/in-memory cache/redis/db

```

Comment:
Why did you replace the call to `ToJson` with a manual call to `JsonSerializer.Serialize` (which is less efficient because it doesn't use the source generator)?